components/Modal/CreateRecipe.js

```diff
@@ -40,16 +42,22 @@ class CreateRecipe extends React.Component {
     const o = Object.assign({}, this.state.recipe);
     o[prop] = e.target.value;
     this.setState({ recipe: o });
+    if (prop === 'name') {
+      this.dismissErrors();
+      this.handleErrors(e.target.value);
```

Comment:
The issue related to displaying the required field message on a field that has focus is still present because `this.handleErrors(e.target.value);` is also calling `this.handleErrorName(recipeName);`.  Should this line be replaced with just `this.handleErrorDuplicate(e.target.value);`?